components/ButtonFooter.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react'
+import ReactNative, { TouchableOpacity, Text, View, Linking } from 'react-native'
+import client, {Color} from '@doubledutch/rn-client'
+
+export class ButtonFooter extends Component {
+    constructor(props) {
+      super()
+    }
+
+    render(){
+      const { buttons } = this.props
+      return(
+          <View style={s.container}>
```

Comment:
Would you mind setting your tab width to 2 for consistency?  `command-comma`, `"editor.tabSize": 2`

Reply:
Set for future updates